polls/templates/polls/auth_token.html

```diff
@@ -21,24 +21,68 @@
         <h3>API Token</h3>
         <code>{{ token }}</code>
       </div>
-      <p class="mt-3">
+
+      <h4 class="mt-3">Admin API</h4>
+      <p>
         Use following code snippet to make request to admin api endpoint.
       </p>
       <pre>
         <code>
 import requests
 headers = {
-    "Authorization": "Token {{ token.key }}"
+    "Authorization":admin_api_url "Token {{ token.key }}"
```

Comment:
Is have any specific reason for adding admin_api_url before Token ?

Reply:
Yeah, there is, the URL is meant to carry the token as an extension